src/cli.js

```diff
@@ -437,12 +438,14 @@ async function build() {
   }
 
   function extractFileGlobs(config) {
```

Comment:
We use this function in both the `getChangedContent.
https://github.com/tailwindlabs/tailwindcss/blob/2ab6e6e30dabe3b3da9ee207e65e751e735d95f9/src/cli.js#L461-L462

But we also use this function when setting up the chokidar `watcher`:
https://github.com/tailwindlabs/tailwindcss/blob/2ab6e6e30dabe3b3da9ee207e65e751e735d95f9/src/cli.js#L702

~And I wonder if this causes issues, because now we are passing normalized paths to chokidar. Do you know if chokidar can handle `C:/../` instead of `C:\..\`? 🤔~

Ah nvm, it fixes chokidar as well: https://github.com/paulmillr/chokidar#api
> Note: globs must not contain windows separators (\), because that's how they work by the standard — you'll need to replace them with forward slashes (/).


Nice!